src/docToPreviewGenerator.ts

```diff
@@ -149,4 +147,26 @@ export class DocToPreviewGenerator {
 
     return data;
   }
+
+  private strBreak =
+    "************************************************************";
+  private showErrorToolsNotFound(msg: string): void {
+    const errorMsgs: string[] = [
+      "D2 executable not found.",
+      "Make sure the D2 executable is installed and on system PATH.",
+      "https://d2lang.com",
+      `${msg}`,
+    ];
+
+    outputChannel.appendError(this.strBreak);
+    for (const m in errorMsgs) {
+      if (errorMsgs[m]) {
+        outputChannel.appendError(errorMsgs[m]);
+      }
+    }
```

Comment:
```suggestion
    for (const m of errorMsgs) {
      outputChannel.appendError(m);
    }
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#difference_between_for...of_and_for...in